### PR TITLE
Provide guidelines on how-to contribute to this project

### PR DESCRIPTION
**Pull request by TurpIF**
JIRA: EE-1213

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +18,17 @@ Please contact your technical support representative for any questions about the
 
 ## License
 [Apache License v2.0](LICENSE)
+
+## Contributing
+
+Any contribution are welcome, as an issue or pull request. 
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
```
Comment (Amokrane) [inline README.md:26]: via issue -> by creating an issue.
I would not require creating an issue for small changes such as: typos, obvious programming mistakes. However, creating an issue is recommended for: API changes, feature changes, bug fixes, improvements etc...

File: README.md [hunk 2]
```diff
@@ -17,3 +18,17 @@ Please contact your technical support representative for any questions about the
...
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your
```
Comment (Amokrane) [inline README.md:29]: note that we have

File: README.md (hunk 1 above)
Comment (TurpIF) [reply README.md]: Reworded

File: README.md [hunk 3]
```diff
@@ -17,3 +18,14 @@ Please contact your technical support representative for any questions about the
 
 ## License
 [Apache License v2.0](LICENSE)
+
+## Contributing
+
+Any contribution is welcome, as an issue or pull request. 
+
+When contributing to this repository, if the changes are not trivial (such as typos or obvious programming mistakes),
```
Comment (Amokrane) [inline README.md:26]: Slightly confusing as I was expecting the example to be related to non trivial changes. I would reword to: "unless the changes are trivial (typos, obvious programming mistakes..), we encourage creating an issue first, to discuss the change you wish to make.

PS: As far as contributing is concerned, I am omitting other channels of communication.

Comment (TurpIF) [reply README.md]: Done
